mixins/form.jsx

```diff
@@ -193,7 +193,7 @@ module.exports = {
     if (errorType === "card_error" && cardError) {
       form.error(cardError.field, cardError.message);
     } else {
-      form.error("other", this.getIntlMessage('try_again_later'));
+      form.error("other", this.getIntlMessage('try_again_later') + " [" + errorType + "]");
```

Comment:
It will be user visible, but the idea is that if someone takes a screenshot we can easily understand the error